hnn_core/tests/test_network.py

```diff
@@ -510,6 +510,25 @@ def test_network_drives_legacy():
                                                  n_bursty_sources)
 
 
+def get_expected_connectivities(net, src_types='all'):
+    """Return expected connectivities left after clearng
+    connections.
+
+    Parameters
+    ----------
+    net - The network instance
```

Comment:
we follow the numpy docstring conventions: https://numpydoc.readthedocs.io/en/latest/format.html#parameters

Reply:
Hey @jasmainak I have simplified the loops and followed the numpy docstring conventions